src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequestMerge.java

```diff
@@ -222,6 +235,15 @@ public void perform(
         }
     }
 
+    private boolean isApproved(GHPullRequest pr) {
+        for (GHPullRequestReview review : pr.listReviews()) {
```

Comment:
your indentation is wrong and will cause checkstyle too fail (hopefully 😃)

Reply:
is fixed